show/main.py

```diff
@@ -1848,9 +1848,9 @@ def ssdhealth(device, verbose, vendor):
 @click.option('-c', '--check', is_flag=True, help="Check the platfome pcie device")
 def pcieinfo(check, verbose):
     """Show Device PCIe Info"""
-    cmd = "pcieutil pcie_show"
+    cmd = "pcieutil pcie-show"
     if check:
-        cmd = "pcieutil pcie_check"
+        cmd = "pcieutil pcie-check"
```

Comment:
Why '-'?

Reply:
Can you check how it is wrking in Intel sonic?

Reply:
@antony-rheneus 

checked in intel board10.28.32.69:
show platform pcieinfo
Usage: pcieutil [OPTIONS] COMMAND [ARGS]...
Try "pcieutil --help" for help.

Error: No such command "pcie_show".
admin@Sonic-L1:~$